tests/stream/test_ffmpegmux.py

```diff
@@ -121,6 +121,33 @@ def run(self):
             ("ffmpegmux", "debug", " bar"),
         ]
 
+    @pytest.mark.parametrize("timeout_value,expected_timeout", [
+        (None, 4.0),  # default
+        (9.5, 9.5),   # custom
+    ])
+    def test_validate_timeout(self, session: Streamlink, timeout_value, expected_timeout):
+        session.options.update({
+            "ffmpeg-no-validation": False,
+        })
+        if timeout_value is not None:
+            session.options.set("ffmpeg-validation-timeout", timeout_value)
```

Comment:
That if-block is unnecessary. Just set the `ffmpeg-validation-timeout` session option in the dict above.